src/main/resources/alma/fix/mediumAndType.fix

```diff
@@ -37,7 +53,10 @@ paste("@leaderTyp+008","@leaderTyp","008", join_char:"")
 # TODO: Check introx concerning possible mappings of 340 https://github.com/hbz/limetrans/blob/d2dff10e1b5cdf5699239a8f3474f8b652d582a3/src/main/resources/transformation/alma.fix#L629
 
 set_array("@300a_mainExtend")
-do list(path:"300  ", "var":"$i")
+do list(path:"300  ", "var":"$iContains fields that describe the subunit and data that identify the host item.
+
+Subunits may be items, folders, boxes, archival series, subgroups, or subcollections.
+")
```

Comment:
There seem to be multiple copy & paste (?) errors. Two more below.

Reply:
Thanks